tests/Spice86.Tests/MachineTest.cs

```diff
@@ -170,7 +169,8 @@ public void TestSegpr(bool enableCfgCpu)
         Machine machine = TestOneBin("segpr", enableCfgCpu);
         if (enableCfgCpu) {
             // Here, a division by 0 occurred causing a CPU fault. It is handled by an interrupt handler.
-            CurrentInstructions currentInstructions = machine.CfgCpu.CfgNodeFeeder.InstructionsFeeder.CurrentInstructions;
+            CurrentInstructions currentInstructions = ((Core.Emulator.CPU.CfgCpu.CfgCpu)machine.Cpu).CfgNodeFeeder
```

Comment:
Can this cast be avoided, like before ?

Reply:
Without adding a type to Machine, unfortunately no. At least not with a lot of effort.

When you look at MachineTest, some tests are toggled off when enableCfgCpu is false.